packages/@aws-cdk/aws-eks/lib/manifest-resource.ts

```diff
@@ -0,0 +1,52 @@
+import cfn = require('@aws-cdk/aws-cloudformation');
+import { Construct, Stack } from '@aws-cdk/core';
+import { Cluster } from './cluster';
+
+export interface KubernetesManifestProps {
+  /**
+   * The EKS cluster to apply this configuration to.
+   *
+   * [disable-awslint:ref-via-interface]
+   */
+  readonly cluster: Cluster;
+
+  /**
+   * The resources in this manifest.
+   */
+  readonly resources: any[];
```

Comment:
looking for some validation later on this? or currently do we just need this to be `any`?

Reply:
At the moment this is opaque but yes, we will be able to provide a rich model here.